<launch>
    <node pkg="nodelet" type="nodelet" name="safe_flight_manager" args="manager" output="screen" />
    
    
    <!-- Pass in sensor_readings_config_file:=/path/to/config.yaml to change options. -->
  <arg name="sensor_readings_config_file"
       default="$(find sensor_readings)/config/sensor_readings.yaml" />
    
    <node pkg="nodelet" type="nodelet" name="sensor_readings_nodelet" args="load sensor_readings/SR safe_flight_manager" output="screen">
        <rosparam file="$(arg sensor_readings_config_file)" command="load" />
    </node>
    
    
    
    
    <node name="laser_scan_broadcaster" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0 /drone /laser 10" />
</launch>
